bitnami/harbor/README.md

```diff
@@ -0,0 +1,275 @@
+
+| Parameter                                                                   | Description                                                                                                                                                                                                                                                                                                                                     | Default                                                 |
+| --------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------- |
+| **Expose**                                                                  |
+| `service.type`                                                               | The way how to expose the service: `ingress`, `clusterIP`, `nodePort` or `loadBalancer`                                                                                                                                                                                                                                                         | `ingress`                                               |
+| `service.tls.enabled`                                                        | Enable the tls or not                                                                                                                                                                                                                                                                                                                           | `true`                                                  |
+| `service.ingress.controller`                                                 | The ingress controller type. Currently supports `default` and `gce`                                                                                                                                                                                                                                                                             | `default`                                               |
+| `service.tls.secretName`                                                     | Fill the name of secret if you want to use your own TLS certificate and private key. The secret must contain keys named `tls.crt` and `tls.key` that contain the certificate and private key to use for TLS. The certificate and private key will be generated automatically if it is not set                                                   |                                                         |
```

Comment:
must contain keys -> must contain two keys named ... , which contain

The certificate and private key will be generated automatically if it is not set -> Will be generated automatically if not set